http-plugins/README.rst

```diff
@@ -5,31 +5,30 @@ URL Plugins
 Introduction
 ============
 
-A Hydrator Real Time Plugin that allows data to be fetched from a given URL at a specified interval.
+A Hydrator real-time plugin that allows data to be fetched from a given URL at a specified interval.
```

Comment:
So i guess this needs to be rewritten a little bit since now this contains multiple plugins (and will contain more in the future). Maybe something like: 
This project is a collection of plugins to perform various HTTP actions in your pipeline. These plugins are currently available:
- HTTPPoller real-time Source
- HTTPCallback